org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/TextEditConverter.java

```diff
@@ -128,6 +132,11 @@ public boolean visit(CopyTargetEdit edit) {
 			Document doc = new Document(compilationUnit.getSource());
 			edit.apply(doc, TextEdit.UPDATE_REGIONS);
 			String content = doc.get(edit.getSourceEdit().getOffset(), edit.getSourceEdit().getLength());
+
+			if (edit.getSourceEdit() != null && edit.getSourceEdit().getSourceModifier() != null) {
```

Comment:
if `edit.getSourceEdit()` was null, it'd have NPE'd the line before. The null check should be done earlier